### ISSUE TITLE
Issue with Int64 values

### ISSUE THREAD

**Issue by Raidus** [roapi/roapi#166]
Hi,

for `Int64` columns I am not able to query the exact value. Following query 
```bash
curl -X POST -d "SELECT *  FROM test WHERE kw_id=2659075375158816300 LIMIT 1" localhost:8080/api/sql
```
gives an empty array.

But when using greater than operator
```bash 
curl -X POST -d "SELECT *  FROM test WHERE kw_id>2659075375158816300 LIMIT 1" localhost:8080/api/sql
```
I'll get results

```json
[
    {
        "kw_id": 7014780707843702634,
        "sv_arr": [1139, 2230, 2709, 3300, 2224, 3195, 3995, 2572, 2503, 1721, 2297, 4685, 2651],
        "date_arr": [
            "2021-02-01",
            "2021-03-01",
            "2021-04-01",
            "2021-05-01",
            "2021-06-01",
            "2021-07-01",
            "2021-08-01",
            "2021-09-01",
            "2021-10-01",
            "2021-11-01",
            "2021-12-01",
            "2022-01-01",
            "2022-02-01"
        ]
    }
]

```

Metadata of parquet file

```json
{
    "test": {
        "fields": [
            { "name": "kw_id", "data_type": "Int64", "nullable": true, "dict_id": 0, "dict_is_ordered": false },
            {
                "name": "sv_arr",
                "data_type": {
                    "List": {
                        "name": "item",
                        "data_type": "Int64",
                        "nullable": true,
                        "dict_id": 0,
                        "dict_is_ordered": false
                    }
                },
                "nullable": true,
                "dict_id": 0,
                "dict_is_ordered": false
            },
            {
                "name": "date_arr",
                "data_type": {
                    "List": {
                        "name": "item",
                        "data_type": "Utf8",
                        "nullable": true,
                        "dict_id": 0,
                        "dict_is_ordered": false
                    }
                },
                "nullable": true,
                "dict_id": 0,
                "dict_is_ordered": false
            }
        ],
        "metadata": {
            "pandas": {
                "index_columns": [{ "kind": "range", "name": null, "start": 0, "stop": 252, "step": 1 }],
                "column_indexes": [
                    {
                        "name": null,
                        "field_name": null,
                        "pandas_type": "unicode",
                        "numpy_type": "object",
                        "metadata": { "encoding": "UTF-8" }
                    }
                ],
                "columns": [
                    {
                        "name": "kw_id",
                        "field_name": "kw_id",
                        "pandas_type": "int64",
                        "numpy_type": "int64",
                        "metadata": null
                    },
                    {
                        "name": "sv_arr",
                        "field_name": "sv_arr",
                        "pandas_type": "list[int64]",
                        "numpy_type": "object",
                        "metadata": null
                    },
                    {
                        "name": "date_arr",
                        "field_name": "date_arr",
                        "pandas_type": "list[unicode]",
                        "numpy_type": "object",
                        "metadata": null
                    }
                ],
                "creator": { "library": "pyarrow", "version": "7.0.0" },
                "pandas_version": "1.4.1"
            }
        }
    }
}

```

Tested with `SQL` and `REST` API. Both behave the same.

Best

**Comment by houqp** [comment]
Interesting, @Raidus do you have a minimal reproducible parquet file that you can share as well?

**Comment by Raidus** [comment]
@houqp sure. Created a minimal file: [Link to parquet file](https://s3.eu-central-1.wasabisys.com/dz-shared/wilhelm/roapi_example_bigint.parquet)

**Comment by houqp** [comment]
Very interesting bug, looks like something is off with datafusion's filter operator, I will dig into it more. Thanks a lot for surfacing the edge-case here!

**Comment by houqp** [comment]
Fix for this landed in master, will be included in the upcoming 0.7.1 release.
